.circleci/config.yml

```diff
@@ -0,0 +1,34 @@
+version: 2
+
+jobs:
+  build:
+    docker:
+      - image: circleci/python:3.6
+    steps:
+      - checkout
+      - restore_cache:
+          keys:
+            - v1-oakcrime-{{ checksum "showCrime/requirements.txt" }}
+            - v1-oakcrime-
+      - run:
+          name: create virtualenv and install dependencies
+          command: |
+            virtualenv venv
```

Comment:
A virtualenv seems unnecessary for an ephemeral build server.  Consider removing this. Is this solely for caching?

Reply:
Yeah, it's for caching. Standard in all of CircleCI's python examples.